sdk/tests/invoiceTypeCode.unit.test.ts

```
@@ -0,0 +1,92 @@
+
+  describe('Explicit type codes', () => {
+    test('emits InvoiceTypeCode 751 when requested', () => {
+      const xml = builder.generateInvoiceXml(buildInput({ invoiceTypeCode: '751' }));
+
+      expect(xml).toContain('<cbc:InvoiceTypeCode>751</cbc:InvoiceTypeCode>');
+      expect(xml).not.toContain('<cbc:InvoiceTypeCode>380</cbc:InvoiceTypeCode>');
+    });
+
+    test.each<InvoiceTypeCode>(['380', '384', '389', '751'])(
+      'accepts CIUS-RO code %s',
+      (code) => {
```

ESLint

[error] Replace `⏎······'accepts·CIUS-RO·code·%s',⏎·····` with `'accepts·CIUS-RO·code·%s',`

(prettier/prettier)